guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -57,6 +77,29 @@ The `filename` option contains the full path to the file that downloads and exec
 |Grub2 UEFI HTTP | `http://_{smartproxy-example-com}_:8000/httpboot/grub2/grubx64.efi` | Requires the `httpboot` feature and renders the `filename` as a full URL where _{smartproxy-example-com}_ is a known host name of {SmartProxy} in {Project}.
 |=======
 
+=== Troubleshooting DHCP Problems in {Project}
+
+{Project} can manage an ISC DHCP server on internal or external DHCP {SmartProxy}. {Project} can list, create, and delete DHCP reservations and leases. However, there are a number of problems that you might encounter on occasions.
+
+.PXE loader option change
+
+When the PXE loader option is changed for an existing host, this causes a DHCP conflict. This is a known issue. Until this issue is fixed, the only workaround is to overwrite the DHCP entry.
+
+.Incorrect permissions on DHCP files
+
+An operating system update can reset the `dhcpd` package. This causes the permissions of important directories and files to reset so that the DHCP {SmartProxy} cannot read required information.
+For more information, see https://access.redhat.com/solutions/2952061[DHCP error while provisioning host from Satellite server Error ERF12-6899 ProxyAPI::ProxyException: Unable to set DHCP entry RestClient::ResourceNotFound 404 Resource Not Found] on Red{nbsp}Hat Knowledgebase.
```

Comment:
I do not think this is fair to add links to Red Hat Knowledgebase for Foreman users because the linked solution can only be accessed by Red Hat users. Maybe add `ifeval` to display this for Satellite only?

Reply:
@lzap  what do you think Foreman user's would think about @spetrosi's point here?

Reply:
In upstream we have https://projects.theforeman.org/projects/foreman/wiki/ERF12-6899

FYI this is a wiki which holds some info for most of our error codes.